estate/models/estate_property.py

```diff
@@ -0,0 +1,32 @@
+from odoo import fields, models
+from dateutil.relativedelta import relativedelta
+
+
+class estate_property(models.Model):
```

Comment:
```suggestion
class EstateProperty(models.Model):
```
Class name should be capitalized.